crates/ruff_python_parser/src/parser/expression.rs

```diff
@@ -731,6 +729,16 @@ impl<'src> Parser<'src> {
                 range: self.node_range(slice_start),
                 parenthesized: false,
             });
+        } else if slice.is_starred_expr() {
+            // If the only slice element is a starred expression, that is represented
+            // using a tuple expression with a single element. This is the second case
+            // in the `slices` rule in the Python grammar.
+            slice = Expr::Tuple(ast::ExprTuple {
```

Comment:
I think it could make sense to copy in the slice rule here because it's unclear to me to which rule you're referring here.

It' something that I've found useful when reading TypeScript's parser code: 
* https://github.com/microsoft/TypeScript/blob/02bb3108ad625cddcec228846b9ff56cc5398976/src/compiler/parser.ts#L4977-L4986
* https://github.com/microsoft/TypeScript/blob/02bb3108ad625cddcec228846b9ff56cc5398976/src/compiler/parser.ts#L5078-L5082

They also often give examples:

* https://github.com/microsoft/TypeScript/blob/02bb3108ad625cddcec228846b9ff56cc5398976/src/compiler/parser.ts#L5186-L5191

Reply:
I see. Thanks for the reference. My only worry with this is that the grammar might get out of sync but it's probably way down the line. I'll add them.

Reply:
I'm thinking of adding this in a future PR clubbed with other document / comment improvements. This way it can be reviewed better.